php/libraries/NDB_Config.class.inc

```diff
@@ -72,10 +72,23 @@ class NDB_Config
                 // in PHP < 5.5, need to assign to a variable first
                 $configFile = $config->configFilePath();
             }
-
             // Load the file
-            $config->load($configFile);
+            //  load config in test
+            $env    = getenv('LORIS_DB_CONFIG');
+            $travis = getenv('TRAVIS');
+            if (!empty($env)) {
+                if (!empty($travis)) {
+                      $file  = dirname(__DIR__);
+                      $file .= $env;
+                } else {
+                    $file = $env;
+                }
+                 $config->load($file);
+            } else {
```

Comment:
Why is this being added here? It should already be handled line ~125 (in configFilePath)

Reply:
@driusan You are right, the first time call this function, it goes ~125. But when a module calls NDB_factory function, it will call NDB_Config.class.inc again, the second time it got an error. this is why I put the fix here.